Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -6752,6 +6801,57 @@ void CClientPed::ResetVehicleInOut()
 //////////////////////////////////////////////////////////////////
 void CClientPed::UpdateVehicleInOut()
 {
+    if (IsLocalEntity())
+    {
+        // If getting inside vehicle
+        if (m_bIsGettingIntoVehicle)
+        {
+            CClientVehicle* pVehicle = GetRealOccupiedVehicle();
```

Comment:
Dont use hungarian notation

Reply:
Fixed

Reply:
Not fixed

Reply:
dont say "fixed" then if you didnt "fix" anything